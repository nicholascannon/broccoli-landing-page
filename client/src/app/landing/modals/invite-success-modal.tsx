import { Button } from '../../../common/components/button';
import { GenericModalView } from '../../../common/modals/components/generic-modal-view';
import { ModalComponent } from '../../../common/modals/modal-engine';

import styles from './invite-success-modal.module.css';

export const InviteSuccessModal: ModalComponent = (props) => {
    return (
        <GenericModalView>
            <div className={styles.content}>
                <h2>All done!</h2>
                <p>You will be one of the first to experience Broccoli &amp; Co. when we launch.</p>
                <Button fullWidth onClick={props.closeModal}>
                    ok
                </Button>
            </div>
        </GenericModalView>
    );
};
