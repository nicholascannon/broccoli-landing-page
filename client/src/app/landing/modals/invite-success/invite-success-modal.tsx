import { Button } from '../../../../common/components/button';
import { Ruler } from '../../../../common/components/ruler';
import { GenericModalView } from '../../../../common/modals/components/generic-modal-view';
import { ModalComponent } from '../../../../common/modals/modal-engine';

import styles from './invite-success-modal.module.css';

export const InviteSuccessModal: ModalComponent = (props) => {
    return (
        <GenericModalView>
            <div className={styles.content}>
                <div>
                    <h2>All done!</h2>
                    <Ruler className={styles.ruler} />
                </div>

                <p>You will be one of the first to experience Broccoli &amp; Co. when we launch.</p>
                <Button fullWidth onClick={props.closeModal}>
                    ok
                </Button>
            </div>
        </GenericModalView>
    );
};
