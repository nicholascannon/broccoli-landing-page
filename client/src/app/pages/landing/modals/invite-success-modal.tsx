import { GenericModalView } from '../../../../common/components/generic-modal-view';
import { ModalComponent } from '../../../../common/modals/modal-engine';

import styles from './invite-success-modal.module.css';

export const InviteSuccessModal: ModalComponent = (props) => {
    return (
        <GenericModalView>
            <div className={styles.content}>
                <h2>All done!</h2>
                <p>Lorem ipsum dolor sit amet consectetur, adipisicing elit. Non, explicabo.</p>
                <button onClick={props.closeModal}>ok</button>
            </div>
        </GenericModalView>
    );
};
