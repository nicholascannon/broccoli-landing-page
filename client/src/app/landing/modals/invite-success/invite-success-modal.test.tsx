/* eslint-disable @typescript-eslint/no-empty-function */
import renderer from 'react-test-renderer';
import { InviteSuccessModal } from './invite-success-modal';

test(`
    GIVEN InviteSuccessModal component
    THEN rendering it should match the snapshot
`, () => {
    const tree = renderer.create(<InviteSuccessModal closeModal={() => {}} />).toJSON();
    expect(tree).toMatchSnapshot();
});
