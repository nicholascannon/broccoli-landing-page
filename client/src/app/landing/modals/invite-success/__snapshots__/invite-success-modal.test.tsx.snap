// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`
    GIVEN InviteSuccessModal component
    THEN rendering it should match the snapshot
 1`] = `
<div
  className="modal"
>
  <div
    className="content"
  >
    <div>
      <h2>
        All done!
      </h2>
      <hr
        className="ruler ruler"
      />
    </div>
    <p>
      You will be one of the first to experience Broccoli & Co. when we launch.
    </p>
    <button
      className="button fullWidth "
      onClick={[Function]}
    >
      ok
    </button>
  </div>
</div>
`;
